docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc

```diff
@@ -1,10 +1,13 @@
 [role="xpack"]
 [testenv="basic"]
 [[search-aggregations-pipeline-inference-bucket-aggregation]]
-=== Inference Bucket Aggregation
+=== {infer-cap} Bucket Aggregation
 
-A parent pipeline aggregation which loads a pre-trained model and performs inference on the
-collated result field from the parent bucket aggregation.
+A parent pipeline aggregation which loads a pre-trained model and performs 
+{infer} on the collated result field from the parent bucket aggregation.
```

Comment:
Should field be plural? Reading this back I'm not sure

on the collated result field**s**